packages/pangea-sdk/tests/integration/asyncio/test_authn.py

```diff
@@ -319,7 +319,7 @@ async def test_authn_z1_user_list(self):
         for user in response.result.users:
             try:
                 await self.authn.user.delete(email=user.email)
-            except pexc.PangeaAPIException:
+            except pe.PangeaAPIException:
                 print(f"Fail to delete user email: {user.email}")
```

Comment:
Same here, should be "Failed', instead of "Fail".

Reply:
fixed